cmake/ext_java.cmake

```diff
@@ -168,36 +161,12 @@ add_custom_command(TARGET onnxruntime_extensions4j_jni POST_BUILD COMMAND ${GRAD
 if (CMAKE_SYSTEM_NAME STREQUAL "Android")
   set(ANDROID_PACKAGE_JNILIBS_DIR ${JAVA_OUTPUT_DIR}/android)
   set(ANDROID_PACKAGE_ABI_DIR ${ANDROID_PACKAGE_JNILIBS_DIR}/${ANDROID_ABI})
-  file(MAKE_DIRECTORY ${ANDROID_PACKAGE_JNILIBS_DIR})
-  file(MAKE_DIRECTORY ${ANDROID_PACKAGE_ABI_DIR})
 
   # Copy onnxruntime_extensions4j_jni.so for building Android AAR package
   add_custom_command(TARGET onnxruntime_extensions4j_jni
-    POST_BUILD COMMAND ${CMAKE_COMMAND}
```

Comment:
Where does these code logic go?

Reply:
A few pieces here:
- calling gradlew with build-android.gradle to build the AAR - this is moved to build_aar.py and done once instead of per ABI
- copying the android test app files, copying the AAR to it, running `gradlew assembleDebug assembleDebugAndroidTest` - this is moved to the CI build. previously we had a separate `gradlew connectedDebugAndroidTest` in the CI build so just moved the rest of the test app setup there